### PR TITLE
[Cloudservice] 0.2.0

### PR DESCRIPTION
**Pull request by qwordy**
Fix `super()` style
Remove `cloud-service role-instance wait`
Add "extended support" in help
Update help of `configuration` parameter
Add confirmation for `cloud-service delete-instance`

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/cloudservice/azext_cloudservice/manual/_help.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
 
 helps['cloud-service'] = """
     type: group
-    short-summary: Manage cloud service
+    short-summary: Manage cloud service (extended support). All commands in this extension are extended support.
```
Comment (sanchitkum) [inline src/cloudservice/azext_cloudservice/manual/_help.py:87]: Should we just keep it as - "Manage cloud service (extended support)."?
The subsequent line doesn't seem to add much context or help clarify.

File: src/cloudservice/azext_cloudservice/manual/_params.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def load_arguments(self, _):
                    help='Specify certificates separated by space that should be installed onto the role instances. '
                    'Format: KeyVaultName:CertificateUrl:CertificateUrl2:...:CertificateUrlN')
         c.argument('configuration', type=file_type, help='Specify the XML service configuration (.cscfg) '
-                   'for the cloud service. Expected value: json-string/@json-file.')
+                   'for the cloud service. Expected value: string/@file.')
```
Comment (sanchitkum) [inline src/cloudservice/azext_cloudservice/manual/_params.py:40]: Can we specify `xml-string/@xml-file`?

File: src/cloudservice/azext_cloudservice/manual/_help.py (hunk 1 above)
Comment (qwordy) [inline src/cloudservice/azext_cloudservice/manual/_help.py:87]:
```suggestion
    short-summary: Manage cloud service (extended support).
```

File: src/cloudservice/azext_cloudservice/manual/_params.py (hunk 2 above)
Comment (qwordy) [inline src/cloudservice/azext_cloudservice/manual/_params.py:40]:
```suggestion
                   'for the cloud service. Expected value: xml-string/@xml-file.')
```

File: src/cloudservice/azext_cloudservice/manual/_help.py (hunk 1 above)
Comment (qwordy) [reply src/cloudservice/azext_cloudservice/manual/_help.py]: Done

File: src/cloudservice/azext_cloudservice/manual/_params.py (hunk 2 above)
Comment (qwordy) [reply src/cloudservice/azext_cloudservice/manual/_params.py]: Done

**Comment by qwordy** [comment]
Azure CLI linter upgraded. We've got new errors. Fixing.
```
===============
| Style Check |
===============

Extensions: cloudservice

Running flake8 on extensions...
Running pylint on extensions...


 Results
=========

Flake8: PASSED

************* Module azext_cloudservice.manual.commands
azure-cli-extensions\src\cloudservice\azext_cloudservice\manual\commands.py:19:4: C0415: Import outside toplevel (azext_cloudservice.generated._client_factory.cf_cloud_service) (import-outside-toplevel)
************* Module azext_cloudservice.generated.commands
azure-cli-extensions\src\cloudservice\azext_cloudservice\generated\commands.py:19:4: C0415: Import outside toplevel (azext_cloudservice.generated._client_factory.cf_cloud_service_role_instance) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\generated\commands.py:41:4: C0415: Import outside toplevel (azext_cloudservice.generated._client_factory.cf_cloud_service_role) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\generated\commands.py:53:4: C0415: Import outside toplevel (azext_cloudservice.generated._client_factory.cf_cloud_service) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\generated\commands.py:77:4: C0415: Import outside toplevel (azext_cloudservice.generated._client_factory.cf_cloud_service_update_domain) (import-outside-toplevel)
************* Module azext_cloudservice
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:22:8: C0415: Import outside toplevel (azure.cli.core.commands.CliCommandType) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:23:8: C0415: Import outside toplevel (azext_cloudservice.generated._client_factory.cf_cloud_service_cl) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:27:17: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:31:8: C0415: Import outside toplevel (azext_cloudservice.generated.commands.load_command_table) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:34:12: C0415: Import outside toplevel (azext_cloudservice.manual.commands.load_command_table) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:41:8: C0415: Import outside toplevel (azext_cloudservice.generated._params.load_arguments) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\__init__.py:44:12: C0415: Import outside toplevel (azext_cloudservice.manual._params.load_arguments) (import-outside-toplevel)
************* Module azext_cloudservice.generated._client_factory
azure-cli-extensions\src\cloudservice\azext_cloudservice\generated\_client_factory.py:13:4: C0415: Import outside toplevel (azure.cli.core.commands.client_factory.get_mgmt_service_client) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\generated\_client_factory.py:14:4: C0415: Import outside toplevel (azure.mgmt.compute.ComputeManagementClient) (import-outside-toplevel)
************* Module azext_cloudservice.manual.custom
azure-cli-extensions\src\cloudservice\azext_cloudservice\manual\custom.py:89:4: C0415: Import outside toplevel (msrestazure.tools.resource_id, msrestazure.tools.is_valid_resource_id) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\manual\custom.py:90:4: C0415: Import outside toplevel (azure.cli.core.commands.client_factory.get_subscription_id) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\manual\custom.py:123:4: C0415: Import outside toplevel (msrestazure.tools.resource_id, msrestazure.tools.is_valid_resource_id) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\manual\custom.py:124:4: C0415: Import outside toplevel (azure.cli.core.commands.client_factory.get_subscription_id) (import-outside-toplevel)
************* Module azext_cloudservice.tests
azure-cli-extensions\src\cloudservice\azext_cloudservice\tests\__init__.py:33:8: C0415: Import outside toplevel (importlib.import_module) (import-outside-toplevel)
azure-cli-extensions\src\cloudservice\azext_cloudservice\tests\__init__.py:55:4: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)

-------------------------------------------------------------------
Your code has been rated at 9.62/10 (previous run: 10.00/10, -0.38)


Pylint: FAILED
```

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**